"uninitialized  variable" warning

Not only on Windows, we got following warning.

See #1490.

```
  CC       parsers/ctags-html.o
parsers/html.c: In function 'readTag':
parsers/html.c:439:6: warning: 'lineOffset_tmp[1]' may be used uninitialized in this function [-Wmaybe-uninitialized]
      makePromise ("JavaScript", startLineNumber, startLineOffset,
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          endLineNumber, endLineOffset, startSourceLineNumber);
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
parsers/html.c:439:6: warning: 'line_tmp[1]' may be used uninitialized in this function [-Wmaybe-uninitialized]
```

--- a/parsers/html.c
+++ b/parsers/html.c
@@ -390,8 +390,8 @@ static bool skipScriptContent (tokenInfo *token, long *line, long *lineOffset)
 	bool found_start = false;
 	bool found_script = false;
 
-	long line_tmp[2];
-	long lineOffset_tmp[2];
+	long line_tmp[2] = {0};
+	long lineOffset_tmp[2] = {0};
 
 	tokenType type;
 

